deploy/backplane/lpsre/acs/02-acs-lpsre-admins-project.ClusterRole.yaml

```diff
@@ -12,3 +12,12 @@ rules:
   - get
   - list
   - watch
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/log
```

Comment:
this is a ClusterRole, this permission is way to broad, if will give us access to pods and logs we are not supposed to. I believe we have to make this permission namespace bound. wdyt?

Reply:
That's fair enough.

Reply:
Sorry, I was wrong, because the SubjectPermission actually creates a RoleBinding to the right Namespaces, not a ClusterRoleBinding, so we are good there. Sorry about the noise!